.eslintrc.js

```diff
@@ -64,11 +88,15 @@ module.exports = {
     },
     {
       files: [
-        "src/renderer/**/*.tsx",
+        "**/*.tsx",
       ],
       parser: "@typescript-eslint/parser",
+      plugins: [
+        "unused-imports"
+      ],
       extends: [
         'plugin:@typescript-eslint/recommended',
+        "plugin:react/recommended",
```

Comment:
perhaps adding "consistent use of double|single quotes" rule in another PR https://eslint.org/docs/rules/quotes
these two lines with different style of quotes doesn't looks sexy